README.md

```diff
@@ -0,0 +1,30 @@
+# terraform-aws-ecr-repository
+A Terraform module to create an Amazon Web Services (AWS) Elastic Container Registry (ECR) repository.
+
+## Usage
+
+```hcl
+module "ecr-repository" {
+  source  = "azavea/ecr-repository/aws"
+  version = "0.1.0"
+
+  repository_name = "my-containers"
+  
+  attach_lifecycle_policy = true
+  max_number_of_images = 7 # expire untagged images older than 7 days
+}
+```
+
+## Variables
+
+- `repository_name` - name of the repository
+- `attach_lifecycle_policy` - attach a lifecycle policy (default: `false`)
+- `lifecycle_policy` - lifecycle policy JSON (default: `${file("${path.module}/templates/default-lifecycle-policy.json.tpl")}`)
```

Comment:
It might be worth clarifying the default a bit here. If I'm familiar with `file()`, I have a sense for what the default is, but if I don't it could cause confusion.

Perhaps something that states that the contents of `default-lifecycle-policy.json.tpl` will be the lifecycle policy that is used.

Reply:
We should also include a link to the ECR lifecycle policy documentation.

Reply:
👍 , I wanted to have a dialogue about the documentation. 

That sounds good and is a good catch. Yes, you pass the contents of a file (or a string literal) to this input.